README.md

```diff
@@ -49,25 +49,25 @@ To run test, run the check command:
 The license is currently set as the GPLv3.0+ as released by the FSF.
 
 This license is compatible with the OpenSSL ASL2.0 license and is a strong
-copyleft license which we find useful.
+copyleft license, which we find useful.
 
 Unlike other copyleft projects we are not dogmatic and chose this license
-for the benefits we think it will brings to a self-contained project like
+for the benefits we think it will bring to a self-contained project like
 kryoptic. Namely that it strongly encourages modifications to be
 contributed back.
 
-If a party asks for it we will pragmatically evaluate a different license
-and will be open to make a change if we think that such change would in fact
+If a party asks for it, we will pragmatically evaluate a different license
+and will be open to make a change if we think that such a change would in fact
 be in the best interest of the project. Note that requests of this kind
-need to come with a well reasoned rationale that shows benefits both for
+need to come with a well-reasoned rationale that shows benefits both for
 the requesting party and the upstream project.
 
 
 # Contributions
 
-Contributions to the project are made under the project's [License][License]
+Contributions to the project are made under the project's [License](License)
```

Comment:
this is still a dead link. I think it should point to LICENSE.txt here: https://github.com/latchset/kryoptic/blob/main/LICENSE.txt

Reply:
I didn't check the link's validity, I just noticed that the Markdown was wrong :D.  Good catch, I've updated the PR!

Reply:
I thought that updating it just to relative link as LICENSE.txt should do, but I did not try that

Reply:
Sorry, I misunderstood your suggestion.  Should I create a new PR to use a relative link?

Reply:
That would be my preference if it will work :)
I wanted @simo5 to have a look into this too as this was mostly what he wrote here.

Reply:
The link from

https://github.com/nwalfield/kryoptic/tree/README

now points to

https://github.com/nwalfield/kryoptic/blob/README/LICENSE.txt

which looks correct.